src/legacyTest/java/org/embulk/util/config/legacy/TestConfigMapper.java

```diff
@@ -19,15 +19,21 @@
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import javax.validation.Validation;
+import javax.validation.constraints.Max;
```

Comment:
Did you run `./gradle checkstyleMain` and `checkstyleTest`? I'm not sure why this PR does not trigger GitHub Actions CI (it's to be fixed aside), but if the CI runs, I expect it would fail.

You have permissions to push your branch directly to `embulk/embulk-util-config`. I guess a direct push would trigger CI...

Reply:
Did run checkstyle, it passed fine, is it failed on your local?
But anyway, I didn't know I'm a collaborator. Will close this fork PR and make a direct one.